assets/scss/_styles_project.scss

```diff
@@ -1,13 +1,31 @@
 @import 'td/code-dark';
 
-
 .td-navbar {
-    .navbar-brand {
-      svg {
-        height: 38px;
-      }
-      .navbar-brand__name {
-        display: none;
-      }
+  .navbar-brand {
+    svg {
+      height: 38px;
+    }
+    .navbar-brand__name {
+      display: none;
     }
   }
+}
+
+.td-home {
+  .cncf {
+    text-align: center;
+
+    p {
+      font-size: 1.2rem;
+      margin-bottom: 0;
+    }
+
+    img {
+      width: 20rem;
+      padding-top: 1rem;
+      max-width: 80%;
+    }
+  }
+}
+
+$tuf-blue: #0082ca;
```

Comment:
This variable isn't used. Can it be dropped, or were you planning on using it later?

Reply:
> This variable isn't used. Can it be dropped, or were you planning on using it later?

I'll use it later. @lukpueh wanted to include the current brand blue color on the [TUF website ](https://theupdateframework.io/) to the New website.